test/e2e_test/endtoend_test.go

```diff
@@ -0,0 +1,101 @@
+package e2e_test
```

Comment:
Let's simplify this dir/file structure and 
1. create a single `e2e` directory at the root of the repo containing these new assets
2. call this test file `e2e_test.go`